### ISSUE TITLE
Issue with Time travel debug

### ISSUE THREAD

**Issue by heckslam** [badoo/MVIKotlin#101]
When I try to connect to device, always receive the error: Error forwarding port via ADB. Make sure there is only one device connected. "
reproduced on emulator and real device. 
`adb devices` shows only one device connected 
OS: Mac os catalina
Plugin version: 2.0.0-rc1
Android studio version: 4.0.0, 4.1.0 beta 3

**Comment by arkivanov** [comment]
Thanks, I will check this! Would you mind posting the full output of the `adb devices` command?

**Comment by heckslam** [comment]
sure. 
Real device: 
```
adb devices
List of devices attached
a2cc8712	device
```
Emulator
```
adb devices
List of devices attached
emulator-5554	device
```

**Comment by arkivanov** [comment]
@heckslam Would you mind to connect just one (any) device and provide an output of the following command as well?
`adb forward tcp:6379 tcp:6379; echo $?`

**Comment by heckslam** [comment]
the output:
```
6379
0
```

**Comment by heckslam** [comment]
for the second time it just outputs zero

**Comment by arkivanov** [comment]
Thanks! So everything is fine with `adb`. I suspect you chose `adb` incorrectly at the very beginning. Could you please try the following: 

- uninstall the plugin
- restart IDE
- install the plugin again
- restart IDE
- connect a device with an application running the `TimeTravelServer` (e.g. the [sample Android app](https://github.com/arkivanov/MVIKotlin/tree/master/sample/todo-app-android))
- click on the following button: ![image](https://user-images.githubusercontent.com/26204457/87253807-e6c35480-c475-11ea-8149-7b90dc5a39a4.png)
- there should be a file chooser dialog, please select `adb` executable (normally it should be located in Users/<user_folder>/Library/Android/sdk/platform_tools)

Please let me know how it works.

**Comment by heckslam** [comment]
I reinstalled the plugin, but it still shows the error. Even tried to clear cache of plugins. file chooser dialog does not show up

**Comment by arkivanov** [comment]
@heckslam This is a very interesting situation. I tried both Android Studio 4.0 and 4.1 Beta 3, on Ubuntu and MacOS Catalina. The following file chooser dialog appears when I click on the "Connect" button:
![image](https://user-images.githubusercontent.com/26204457/87258112-e9ae4d00-c455-11ea-991f-d38f3a545854.png)

If you don't mind I would build a special version of the plugin with some debug messages so you could try and provide me with more information.

**Comment by arkivanov** [comment]
Also could you please try the following:

- Open Help | Edit Custom VM Options
- Add the following line `-Dide.mac.file.chooser.native=false`
- Restart IDE
- Try clicking the "Connect" button

**Comment by arkivanov** [comment]
I have also made some fixes and added some meaningful logs. Would be great If you could try the attached build. If the problem still there (no file chooser dialog), could you please attach IDE logs file (Help | Show Log in Finder)?
[plugin-idea-2.0.0-rc1.zip](https://github.com/arkivanov/MVIKotlin/files/4909888/plugin-idea-2.0.0-rc1.zip)

**Comment by heckslam** [comment]
I read the logs myself. I don’t know how, but by default the wrong path to adb was chosen (It threw : _"error=13, Permission denied"_). So I just renamed the directory to that path and the plugin suggested me to select path to adb. File selector dialog showed up and everything began to work great🥳. Thank you so much for your help

**Comment by heckslam** [comment]
I think it would be great to have more meaningful error message for that error instead of port forwarding issue

**Comment by arkivanov** [comment]
@heckslam thanks for the update! I will update the plugin with more meaningful error messages.

**Comment by arkivanov** [comment]
Fixed via #105 and #107
